ligo/stablecoin/actions.ligo

```diff
@@ -146,6 +146,91 @@ function debit_from
   end
 } with store with record [ ledger = updated_ledger ]
 
+(*
+ * Converts a transfer item into a safelist transfer item
+ *)
+function convert_to_safelist_transfer
+  (const tp : transfer_param
+  ) : safelist_transfer_item is
+    record
+      [ from_ = tp.0
+      ; to_ = tp.1.0
+      ]
+
+(*
+ * Calls `assert_transfer` of the provided safelist contract using
+ * the provided transfer_descriptors.
+ *)
+function call_assert_transfers
+  ( const ops_in : list(operation)
+  ; const opt_sl_address : option(address)
+  ; const transfer_params : transfer_params
+  ) : list(operation) is block
+  { const operations: list(operation) =
+      case opt_sl_address of
+        Some (sl_address) ->
+          case (Tezos.get_entrypoint_opt ("%assertTransfers", sl_address) : option(contract(safelist_assert_transfers_param))) of
```

Comment:
entrypoints should be in `snake_case` form

Reply:
I just followed what is written here https://github.com/tqtezos/stablecoin/blob/master/docs/specification.md#safelist

Reply:
Oh, apparently this needs to be updated

Reply:
Why though? Is it part of FA2?

Reply:
No, but we need to follow their guidelines and it's confusing having both cases in entrypoints call, apparently in future some ligo dev may decide to only enforce one structure which can actually happen considering what's happening right now with specification

Reply:
Recently whitelist interface was proposed to TZIP: https://gitlab.com/tzip/tzip/-/merge_requests/61/diffs

And our contract should work with https://github.com/tqtezos/lorentz-contract-whitelist/

So existing names (`CamlCase`) are fine.